### ISSUE TITLE
Failed to initialize

### ISSUE THREAD

**Issue by DaveO777** [MaFi-Games/Captain-of-Industry-issues#229]
**Describe the issue**
Installed and Ran game yesterday - about 10 hours with autosave and manual save. 
Started the game today and 'continue' button was greyed out and the save files were missing from the 'load' menu.
Attempted to start a new game and received a 'failed to initialize' message

**Expected behavior**
Save file / continue previous session.
Alternatively new game load.

**To Reproduce**
Start and run game.

**Additional context**
 - Game version: EA 0.4.1E

**Attachments**
Log files attached.
[22-06-06_13-30-34_2222.log](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/8841742/22-06-06_13-30-34_2222.log)
[22-06-06_13-29-05_2714.log](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/8841743/22-06-06_13-29-05_2714.log)

**Comment by MaFi-Games** [comment]
I see this in errors: "Could not find file "C:\Users\doake\OneDrive\Documents\Captain of Industry\Console""
In the meantime. Could you please make sure that directory exists and try it again?

**Comment by DaveO777** [comment]
I found there are 2 separate save directories, One in "Docs" and one in "steamapps"

<img width="620" alt="image" src="https://user-images.githubusercontent.com/106942991/173208511-d94cc770-f7c4-4777-ac82-4fc4f1f70842.png">
<img width="607" alt="image" src="https://user-images.githubusercontent.com/106942991/173208523-85d01702-3509-412e-879b-8a953e549f42.png">

Currently, I have the save files available from the "Docs" available in the game.
<img width="363" alt="image" src="https://user-images.githubusercontent.com/106942991/173208584-22f3c3e9-fefa-4868-be7a-3d0ab3656cb0.png">


This doesn't appear to be related to the 'failed to initialize' message. I had to verify game files in steam to allow the game to run again. 
BTW I got the issue above on my first run of the game.
